06-Advanced_Topics_and_Plans/06-Advanced_Topics_and_Plans.ipynb

```diff
@@ -0,0 +1,509 @@
+{
```

Comment:
<p>Enumerate the two ways as a numbered list?</p><p> </p><p>Something is messed up with <code>Globally selecting inline storage</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>